src/components/Navigation.js

```diff
@@ -32,6 +37,14 @@ function Navigation() {
     helpers.openExternalURL(helpers.getExplorerURL());
   }
 
+  const renderNCNavigation = () => {
+    return (
+      <li className="nav-item">
+        <NavLink to="/nano_contract/" exact className="nav-link" activeClassName="active" activeStyle={{ fontWeight: 'bold' }}>{t`Nano Contract`}</NavLink>
```

Comment:
chore: Some of those props are no longer allowed since the upgrade to React Router v6. You can see which ones were removed on [the diff of PR 555](https://github.com/HathorNetwork/hathor-wallet/pull/555/files#diff-4e130ab97c387c481bb920b51e80a88b3148332602174ea833378ad96428d810)

If you just keep the pattern of the other ones in this file, with only `to` and `className`, it should work ok.

Also: the atomic Swap navigation button has a different approach to the conditional rendering, injecting the element directly without using a function. It would be good to keep a consistent pattern of those conditionals.

Reply:
Removed the unecessary attributes and used the same conditional format as in the atomic swap:
https://github.com/HathorNetwork/hathor-wallet/pull/528/commits/46e45f43ca4a980384e084203e0265d1cee2de76